tutorials/plot_background_statistics.py

```diff
@@ -0,0 +1,566 @@
+# values by taking random subsets of samples (subjects), flipping the sign
+# of their data, and recording the resulting statistic absolute value. The
+# absolute value of the statistic evaluated on the veridical data can then be
+# compared to this distribution, and the p-value is simply the proportion of
+# null distribution values that were smaller.
+#
+# .. note:: In the case where ``n_permutations`` is large enough (or "all") so
+#           that the complete set of unique resampling exchanges can be done
+#           (which is :math:`2^{N_{samp}}-1=1023` for the one-tailed paired
+#           test here, not counting the veridical distribution),
+#           instead of randomly exchanging conditions the null is formed
+#           from using all possible exchanges. This is known as a permutation
+#           test (or exact test) form of a non-parametric resampling test.
+
+# Here we have to do a bit of gymnastics to get our function to do
+# a permutation test without correcting for multiple comparisons:
+
+# Let's flatten the array for simplicity
+X.shape = (n_subjects, n_src)
+titles.append('Permutation')
+ts.append(np.zeros(width * width))
+ps.append(np.zeros(width * width))
+mccs.append(False)
+for ii in range(n_src):
+    ts[-1][ii], ps[-1][ii] = \
+        permutation_t_test(X[:, [ii]], verbose=True if ii == 0 else False)[:2]
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# Multiple comparisons
+# --------------------
+# So far, we have done no correction for multiple comparisons. This is
+# potentially problematic for these data because there are
+# :math:`40 \times 40 = 1600` tests being performed. If we just use
+# a threshold ``p < 0.05`` for all of our tests, we would expect many
+# voxels to be declared significant even if there were no true effect.
+# In other words, we would make many **type I errors** (adapted from
+# `Wikipedia <https://en.wikipedia.org/wiki/Type_I_and_type_II_errors>`_):
+#
+# .. rst-class:: skinnytable
+#
+#   +----------+--------+------------------+------------------+
+#   |                   |          Null hypothesis            |
+#   |                   +------------------+------------------+
+#   |                   |       True       |       False      |
+#   +==========+========+==================+==================+
+#   |          |        | Type I error     | Correct          |
+#   |          | Yes    |   False positive |   True positive  |
+#   + Reject   +--------+------------------+------------------+
+#   |          |        | Correct          | Type II error    |
+#   |          | No     |   True Negative  |   False negative |
+#   +----------+--------+------------------+------------------+
+#
+# To combat this problem, multiple methods exist. Typically these
+# provide control over either the:
+#
+# 1. `Familywise error rate (FWER) <fwer>`_
+#      The probability of making one or more type I error:
```

Comment:
error(s)

Reply:
Also, I think this is not perfectly accurate. This is all conditional on the value of H0. I.e., the probability of rejecting a H0 when etc etc. Right?

Reply:
Simplest definition fits the one here

https://en.wikipedia.org/wiki/Family-wise_error_rate

Reply:
Doesn't mean it's correct ... As you know, frequentist p is always conditional on H0. I.e., p is the probability under H0.
Phrased differently, if all H0s are false, the probability of making any type I errors is zero. So I don't think this definition is correct, the correct definition must somehow condition on H0.

Reply:
Well it's not just there, I see it this way many places (e.g., http://www.gs.washington.edu/academics/courses/akey/56008/lecture/lecture10.pdf). I think it's a matter of the level of detail we want to go into. I'd prefer to keep it simple for people, but if you think the conditional will add to people's understanding or prevent them from making errors I can add it.